x-pack/plugins/security_solution/public/timelines/components/manage_timeline/index.tsx

```diff
@@ -91,7 +99,16 @@ const reducerManageTimeline = (state: ManageTimelineById, action: ActionManageTi
           ...state[action.id],
           ...action.payload,
         },
-      };
+      } as ManageTimelineById;
+    case 'SET_INDEX_TO_ADD':
+      const timelineIdState: ManageTimeline = state[action.id] as ManageTimeline;
```

Comment:
nit: consider dropping the `Id` from `timelineIdState`, because this variable holds the state of a timeline, as opposed to the state of an identifier